apis/apps/v1alpha1/clusterdefinition_types.go

```diff
@@ -823,55 +821,20 @@ func (r *ReplicationSetSpec) FinalStsUpdateStrategy() (appsv1.PodManagementPolic
 	return appsv1.ParallelPodManagement, s
 }
 
-type SwitchPolicy struct {
-	// switchPolicyType defines type of the switchPolicy.
-	// MaximumAvailability: when the primary is active, do switch if the synchronization delay = 0 in the user-defined lagProbe data delay detection logic, otherwise do not switch. The primary is down, switch immediately.
-	// MaximumDataProtection: when the primary is active, do switch if synchronization delay = 0 in the user-defined lagProbe data lag detection logic, otherwise do not switch. If the primary is down, if it can be judged that the primary and secondary data are consistent, then do the switch, otherwise do not switch.
-	// Noop: KubeBlocks will not perform high-availability switching on components. Users need to implement HA by themselves.
-	// +kubebuilder:validation:Required
-	// +kubebuilder:default=MaximumAvailability
-	Type SwitchPolicyType `json:"type"`
-
-	// switchStatements defines switching actions according to their respective roles, We divide all pods into three switchStatement role={Promote,Demote,Follow}.
-	// Promote: candidate primary after elected, which to be promoted
-	// Demote: primary before switch, which to be demoted
-	// Follow: the other secondaries that are not selected as the primary, which to follow the new primary
-	// if switchStatements is not set，we will try to use the built-in switchStatements for the database engine with built-in support.
-	// +optional
-	SwitchStatements *SwitchStatements `json:"switchStatements,omitempty"`
-}
-
-type SwitchStatements struct {
-	// promote defines the switching actions for the candidate primary which to be promoted.
-	// +optional
-	Promote []string `json:"promote,omitempty"`
+type SwitchoverSpec struct {
+	CommandExecutorEnvItem `json:",inline"`
```

Comment:
don't inline, keep consistent struct and naming patterh with `SystemAccountSpec`